dev-requirements.in

```diff
@@ -1,5 +1,6 @@
 -c requirements.txt
 
+click>= 8
```

Comment:
You sure you want to support any greater than 8? Like version 9 when it comes out? Or do you want `~=8.0`?